test/io/output/output_test.py

```diff
@@ -90,3 +90,6 @@ def testTestrun(self):
   def testUpdateOutput(self):
     """Make sure we don't accidentally leave UPDATE_OUTPUT True."""
     assert not self.UPDATE_OUTPUT, 'Change UPDATE_OUTPUT back to False!'
+
+if __name__ == '__main__':
```

Comment:
hm, i don't think we include this by convention, because we run the tests with pytest via `python setup.py test`.  It does make the test more standalone-runnable, though.  let's leave it out for now, and if you really like having it there, we can talk about adding this to all the unit tests.